pimd/pim_cmd.c

```diff
@@ -5769,6 +5798,52 @@ DEFUN (show_ip_multicast_vrf_all,
 	return CMD_SUCCESS;
 }
 
+DEFUN(show_ip_multicast_count, show_ip_multicast_count_cmd,
+      "show ip multicast count [vrf NAME] [json]",
+      SHOW_STR IP_STR
+      "Multicast global information\n"
+      "Data packet count\n" VRF_CMD_HELP_STR JSON_STR)
+{
+	int idx = 2;
+	struct vrf *vrf = pim_cmd_lookup_vrf(vty, argv, argc, &idx);
+	bool uj = use_json(argc, argv);
+
+	if (!vrf)
+		return CMD_WARNING;
+
+	show_multicast_interfaces(vrf->info, vty, uj);
+
+	return CMD_SUCCESS;
+}
+
+DEFUN(show_ip_multicast_count_vrf_all, show_ip_multicast_count_vrf_all_cmd,
+      "show ip multicast count vrf all [json]",
+      SHOW_STR IP_STR
+      "Multicast global information\n"
+      "Data packet count\n" VRF_CMD_HELP_STR JSON_STR)
+{
+	bool uj = use_json(argc, argv);
+	struct vrf *vrf;
+	bool first = true;
+
+	if (uj)
+		vty_out(vty, "{ ");
```

Comment:
I'm just curious: why this way? I mean why not using normal JSON-like operations we have?

Reply:
I just followed the other commands in pim_cmd.c